src/mqt/bench/benchmark_generation.py

```diff
@@ -219,7 +219,7 @@ def get_indep_level(
 @overload
 def get_native_gates_level(
     qc: QuantumCircuit,
-    gateset: Gateset,
+    gateset: Target,
```

Comment:
I'd simply refer to this as `target` everywhere.

Reply:
Agreed and adjusted it. However, for the `get_benchmark` method I think it is more easily understandable to still call it `gateset`. I would like to discuss this and some other points in person.